### DISCUSSION TITLE
Wanted to contribute

### DISCUSSION THREAD

**Discussion by SkaarFacee** [mindee/doctr#1368] [General]
Hey, I wanted to contribute to this amazing library! 
I was thinking of starting off with #524 or #1007. In order to have a discussion regarding the same, I wanted to join the slack channel however I did not have access to it. I was hoping to get some direction regarding how I could help on one of those issues.

**Comment by felixT2K** [comment]
Hi @SkaarFacee 👋🏼,

Great to hear that you want to help us it's highly apprecated.

Does this invitation link work for you: https://join.slack.com/t/mindee-community/shared_invite/zt-260fq9g9z-5fh6Saztxet4wLUMfv1MuQ 
?

**Reply by SkaarFacee** [reply]
Yes this invitation link worked.
Thank you so much :)

**Comment by felixT2K** [comment]
About the mentioned issues both are huge topics i suggest to start maybe with some smaller tasks to get in touch with the library ? wdyt ?

#610
#924 
#972

NOTE:
The plan is not to add an additional model before the next release is done :)
(If you still want to work on something like this then we can plan it but we would have to keep it as a draft until the next release is through)
I know that  @nikokks is interested in adding https://github.com/czczup/FAST

**Reply by SkaarFacee** [reply]
Okay, I totally understand. I will look one of the issues you have mentioned and start off with that. 
PS: @nikokks Do we have an GH issue about adding FAST ?
